go.mod

```diff
@@ -28,17 +28,17 @@ require (
 	golang.org/x/crypto v0.0.0-20220411220226-7b82a4e95df4
 	golang.org/x/text v0.3.7
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.24.2
-	k8s.io/apiextensions-apiserver v0.24.2
-	k8s.io/apimachinery v0.24.2
+	k8s.io/api v0.25.0
```

Comment:
I gather this is not a problem, right? ( It's a minor version jump which might cause some issues with dependent libraries ).

I guess if all tests passed this shouldn't be a problem.

Reply:
I agree since we haven't bumped our test versions yet, it should be 0.24.x and CAPI 1.2.2 is using it.

Reply:
Since we are using this `k8s.io/component-base/logs/api/v1`, it pulls v0.25.x k8s.io dependencies.
Reverted back to 0.25.x.